### DISCUSSION TITLE
Calendar default view to seven days

### DISCUSSION THREAD

**Discussion by sistemasnow** [YetiForceCompany/YetiForceCRM#14641] [How to] [answered]
Dear all,

¿Is there anyway to see all days in calendar as default option? 

Maybe, the second idea to do this would be to set the working days from monday to sunday if that´s possible.

Thanks in advance and best regards.

Yeti version: 6.1.0

**Comment by vovpff** [comment, marked as answer]
Сheck /config/modules/Calendar.php

**Reply by sistemasnow** [reply]
Thanks for the answer @vovpff.

It worked!

Best regards.
